es5-shim.js

```diff
@@ -6,12 +6,15 @@
 
 // vim: ts=4 sts=4 sw=4 expandtab
 
-// Add semicolon to prevent IIFE from being passed as argument to concatenated code.
-;
+// Added to override editor options
+/*jshint maxparams:false, maxdepth:false, maxstatements:false, maxcomplexity:false,
```

Comment:
I know, it was just while in my editor, I will remove stuff that has nothing to do with the fix.